src/pages/wallets/find-wallet.tsx

```diff
@@ -168,7 +168,7 @@ const FindWalletPage = ({ data, location }) => {
         top="76px"
         bg="background.base"
         w="full"
-        zIndex={1}
+        zIndex={11}
```

Comment:
oh did not know about this. this case would be ` sticky: 1100,` ?

Reply:
Just realized we can't do it because the header is z-index:100, and it's sticky.
we can't follow the chakra list unless all components do follow these rules.
changed to 20 here and we will follow on this later

Reply:
@nloureiro Fyi, our Chakra-UI zIndex tokens:
```
hide: -1
auto: auto
base: 0
docked: 10
dropdown: 1000
sticky: 1100
banner: 1200
overlay: 1300
modal: 1400
popover: 1500
skipLink: 1600
toast: 1700
tooltip: 1800
```